qoute/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>  
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">
+</head>
+<body>
+    <main class="index-page">
+        <!-- header section -->
+        <header>
+            <nav class="nav">
+                <h3 class="logo">OurLibrary</h3>
+                <ul>
+                    <a href="#"><li>Home</li></a>
+                    <li id="appear-about">About</li>
+                </ul>
+            </nav>
+        </header>
+        <!-- header section end -->
+        <!-- about section -->
+        <section id="about" class="about">
+            <button id="closeBtn">&times;</button>
+            <article class="about-content">
+            <h3>THE IMPORTANCE OF READING IN OUR LIFE</h3>
+            <p>Reading is important for many reasons. Firstly, it develops our thoughts. Secondly, it gives us endless knowledge and lessons. Thirdly, it keeps our minds active. Finally, books can hold and keep all kinds of information, stories, thoughts and feelings unlike anything else.
+            </p><br>
+            <p id="sec-p">So we have built this website to give you the opportunity to discover an intersting world.</p>
+            </article>
+        </section>
+        <!-- about section end -->
+
+
+ <!-- header section  -->
+    <div class="quote_header">
+        <h1 class="quote_titel">Quotations</h1>
+        <p class="p_titel">
+            Small pieces from wide world
+        </p>
+    </div>
+
+    <!-- Quote section start -->
+    <div class="quote_generator" id="quote_generator">
+        <!-- Quote -->
+        <div class="quote_text">
+             <i class="fas fa-quote-left"></i>
+             <span id="quote"></span>
+        </div>
+        <!-- Author -->
+        <div class="quote_author">
+            <span id="author"></span>
+        </div>
+        <!-- Buttons -->
+        <div class="button_container">
+            </button>
+            <button id="new_quote">New Quote</button>
+        </div>
+    </div>
+    <!-- Quote section end -->

```

Comment:
**As the previous comment 👆**

**use the semantic element `section` here to wrap this part instead of `div`**